### ISSUE TITLE
[AlertDialog] Content doesn't have the 'alertdialog' aria-role

### ISSUE THREAD

**Issue by peloycosta** [reach/reach-ui#431]
### Current Behavior

Right now the AlertDialogContent has the role 'dialog'

Note: The AlertDialog examples in the website have the correct aria-role.

### Expected behavior

AlertDialogContent should have aria-role='alertdialog' , according to [WAI ARIA](https://www.w3.org/TR/wai-aria-practices-1.1/#alertdialog)

### Reproducible example

The example in the alert-dialog package reproduces this issue. [example](https://github.com/reach/reach-ui/blob/master/packages/alert-dialog/examples/basic.example.tsx)

### Suggested solution(s)

One solution could be to move the DialogContent role above the prop spread [here](https://github.com/reach/reach-ui/blob/2f0008223b6ffd6764c864bd7a20fd9adac0ae24/packages/dialog/src/index.tsx#L166)

**Comment by chancestrickland** [comment]
Fixed in 0.7.4!
